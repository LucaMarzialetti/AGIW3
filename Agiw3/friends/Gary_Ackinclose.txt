--url_fb https://www.facebook.com/profile.php?id=100011009292191
--friends
VM6931:3 Cindy Robinson
VM6931:3 Craig Beauman
VM6931:3 Lee Fiddes
VM6931:3 Neil Ackinclose
VM6931:3 Nina Berry
VM6931:3 Cheryl Beaumont-Harrison
VM6931:3 Neil Fraser
VM6931:3 Victoria Bartle
VM6931:3 Janet Fay
VM6931:3 Phil Michael Pugh
VM6931:3 David SkonZac Williams
VM6931:3 Janette Johnson
VM6931:3 Paul James Reynolds
VM6931:3 Sharon Watson
VM6931:3 Kate Speed
VM6931:3 Alfie Bailey
VM6931:3 Christopher Mitford
VM6931:3 Mandy Smith
VM6931:3 Nicola Emerson
VM6931:3 Caroline Parkinson
VM6931:3 Gemma Moore
VM6931:3 Robert Doyle
VM6931:3 Daniel Evans
VM6931:3 Joshua Riffe
VM6931:3 Kristine Louise Zuliani
VM6931:3 Jayne Middleton Potter
VM6931:3 Quentin Caisley
VM6931:3 Nick Barnard
VM6931:3 Peter Paterson
VM6931:3 Michael Alan Smith
VM6931:3 The Commissioners Quay Inn
VM6931:3 Newcastle United