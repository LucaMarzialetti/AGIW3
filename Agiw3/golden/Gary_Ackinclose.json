{
    "PATTERN": {
        "tel": [
            "0191 296 1777",
            "0191 257 9326"
        ],
        "addr": ["NE30 1QQ"],
        "email": ["john.inman@hindle"]
    },
    "NER": {
        "LOC": [
            "England",
            "Wales"
        ],
        "ORG": [
            "Hindle Campbell Law",
            "HC Law LLP",
            "Employment Tribunals",
            "Solicitors Regulation Authority",
            "Safety Tribunal",
            "Northumberland Square"
        ],
        "PER": [
            "Hindle Campbell",
            "Court representation Mediation representation Directors",
            "Cristina Falzon",
            "John Inman"
        ]
    },
    "url": "http://www.hindle-campbell.co.uk/employment.html"
}